### PR TITLE
Engagement container and several icons

### PR DESCRIPTION
**Pull request by RavshanovUsmonbek**
1. Engagement container component created
2. Several icons were added
3. Filtering component was added

### CODE REVIEW

File: static/assets/js/vue_components/FIlterToolbarComponent.js
Comment (Aspect13) [inline static/assets/js/vue_components/FIlterToolbarComponent.js:1]: check if it is the right place for this

**Comment by Aspect13** [comment]
@RavshanovUsmonbek If theese components are only used in one of your plugins then maybe this is not the right place for them. The idea behind vue_components is that we place common base components to build pages with like buttons, dropdowns, etc.

File: static/assets/js/vue_components/FIlterToolbarComponent.js
Comment (RavshanovUsmonbek) [reply static/assets/js/vue_components/FIlterToolbarComponent.js:1]: I will check its usage one more time, if it is used only in one place then I will remove it from here

File: static/assets/js/vue_components/EngagementWrapper.js
[the remark this answers is not in the document]
Comment (RavshanovUsmonbek) [reply static/assets/js/vue_components/EngagementWrapper.js:1]: It is used in several places, however, I think I can move it to engagements plugins and import it from there into other plugins.

**Comment by RavshanovUsmonbek** [comment]
@Aspect13 FYI: I removed both filter and engagement container components

**Pull request closed without merging**